### PR TITLE
[docs]: standardize vectorstores

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py [hunk 1]
```diff
@@ -26,15 +26,106 @@
...
+                # other params...
+            )
+
+    # TODO: Populate with relevant variables.
+    Add Documents:
+        .. code-block:: python
+
+            from langchain_core.documents import Document
+
+            document = Document(page_content="foo", metadata={"baz": "bar"})
```
Comment (baskaryan) [inline libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py:74]: can we put the id on the document directly (is that supported in all vecstores @eyurtsev?)

File: libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py [hunk 2]
```diff
@@ -26,15 +26,106 @@
...
+    Delete Documents:
+        .. code-block:: python
+
+            vector_store.delete(ids=["1"])
+
+    # TODO: Fill out with relevant variables and example output.
+    Search:
+        .. code-block:: python
+
+            results = vector_store.similarity_search(query="thud",k=1,filter={"source":"https://example.com"})
```
Comment (baskaryan) [inline libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py:97]: not all vecstores support filtering, and even for ones that do there's not a standard filter syntax. so would drop that part

File: libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py [hunk 3]
```diff
@@ -26,15 +26,106 @@
...
+
+        .. code-block:: python
+
+            # TODO: Example output
+
+    # TODO: Fill out with relevant variables and example output.
+    Use as Retriever:
+        .. code-block:: python
+
+            retriever = vector_store.as_retriever()
```
Comment (baskaryan) [inline libs/cli/langchain_cli/integration_template/integration_template/vectorstores.py:121]: would should search_kwargs= being passed in

File: libs/cli/langchain_cli/integration_template/docs/vectorstores.ipynb [hunk 4]
```diff
@@ -17,35 +17,88 @@
...
+   "cell_type": "markdown",
+   "id": "9695dee7",
+   "metadata": {},
+   "source": [
+    "### Credentials\n",
+    "\n",
+    "- TODO: Update with relevant info.\n",
     "\n",
-    "## Installation"
+    "Head to (TODO: link) to sign up to __ModuleName__ and generate an API key. Once you've done this set the __MODULE_NAME___API_KEY environment variable:"
```
Comment (leo-gan) [inline libs/cli/langchain_cli/integration_template/docs/vectorstores.ipynb:57]: Is the logic across API_KEYs conditional? 
We have plenty of vectorstores without API_KEYs.

Comment (efriis) [reply libs/cli/langchain_cli/integration_template/docs/vectorstores.ipynb:57]: For the integrations that don't need one, they can remove this line!

This is just a template for bootstrapping docs with the CLI

**Review by efriis — APPROVED** [review]

**Pull request merged**